lib/dockerregistry/types.go

```diff
@@ -250,6 +251,15 @@ type RegistryContext struct {
 	Src            bool         `yaml:"src,omitempty"`
 }
 
+// GManifestListContext is used only for reading GManifestList information from
```

Comment:
Nit: GCR might be a better prefix than G

Reply:
Ack